sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -37,29 +37,29 @@ The following examples demonstrate how to add the `AzureMonitorExporter` to your
 
 - Traces
     ```csharp
-    Sdk.CreateTracerProviderBuilder()
-        .AddAzureMonitorTraceExporter(o => o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000")
+    var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
We may need to explain, during the application exit dispose `tracerProvider`.
This feedback is applicable for other signals too.

Reply:
done